server/cluster.go

```diff
@@ -54,7 +54,7 @@ func (req *CreateClusterRequest) validate() error {
 
 	invalidNodes := make([]string, 0)
 	for _, node := range req.Nodes {
-		if !util.IsIPPort(node) {
+		if !util.IsIPPort(node) && !util.IsValidDNS(node) {
```

Comment:
How about renaming the IsIPPort to IsHostPort, then checking if the host is an IP or a valid domain name? For checking the domain name, it'd be better to use `net.LookupHost` directly.

Reply:
Great ! did it now